### PR TITLE
Updated social icons

### PR DESCRIPTION
**Pull request by Spnetic-5**
Added links to GitHub and LinkedIn via social icons.

Fixes: #8

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
     "@fortawesome/free-brands-svg-icons": "^5.15.1",
     "@fortawesome/free-solid-svg-icons": "^5.15.1",
     "@fortawesome/react-fontawesome": "^0.1.14",
-    "next": "9.5.5",
+    "next": "^11.0.0",
```
Comment (VedantParanjape) [inline package.json:28]: Don't change this. Revert to older change.

File: components/Footer/Footer.js [hunk 2]
```diff
@@ -5,15 +5,24 @@ import {
...
 import { SocialMediaLinks } from '../../data';
+import { faCode } from '@fortawesome/free-solid-svg-icons';
 
 const Footer = () => {
   return (
     <>
       <footer className={styles.footer}>
         <h3>Connect with us</h3>
         <div className={styles.footerIcons}>
+        <a href={SocialMediaLinks.github} target='_blank'>
```
Comment (VedantParanjape) [inline components/Footer/Footer.js:20]:
```suggestion
         <a href={SocialMediaLinks.github} target='_blank'>
```

Comment (VedantParanjape) [reply components/Footer/Footer.js:20]: Indent with the other blocks.

File: components/Footer/Footer.js [hunk 3]
```diff
@@ -5,15 +5,24 @@ import {
   faTwitter,
   faInstagram,
   faYoutube,
+  faGithub,
+  faLinkedinIn
 } from '@fortawesome/free-brands-svg-icons';
 import { SocialMediaLinks } from '../../data';
+import { faCode } from '@fortawesome/free-solid-svg-icons';
```
Comment (VedantParanjape) [inline components/Footer/Footer.js:12]: move this above `import { SocialMediaLinks } from '../../data';`

**Comment by VedantParanjape** [comment]
@Spnetic-5 Please deploy the site on your own account and post the link here, to verify that it works fine.

File: package.json (hunk 1 above)
Comment (kkothari2001) [reply package.json]: Yes pls, this has caused many issues before.

**Comment by VedantParanjape** [comment]
@Spnetic-5 The test link is still missing.

**Comment by Spnetic-5** [comment]
> @Spnetic-5 Please deploy the site on your own account and post the link here, to verify that it works fine.

https://confident-pasteur-51d4dd.netlify.app/

**Comment by VedantParanjape** [comment]
@Spnetic-5 package-lock.json is still there. Please remove it.

**Comment by VedantParanjape** [comment]
Fixed #9

**Pull request merged**